src/avt/Filters/avtLineToPolylineFilter.C

```diff
@@ -157,28 +159,33 @@ avtLineToPolylineFilter::ExecuteData(avtDataRepresentation *inDR)
     // straight away.
     //
     input->GetLines()->InitTraversal();
-    vtkIdType n, *pts = 0;
+    vtkIdType n = 0;
+#if LIB_VERSION_LE(VTK,8,1,0)
+    vtkIdType *cellPts = nullptr;
+#else
+    const vtkIdType *cellPts = nullptr;
+#endif
     std::set<edge> freeEdges;
     vtkIdType toCellId = input->GetVerts()->GetNumberOfCells();
-    for(vtkIdType cellid = 0; input->GetLines()->GetNextCell(n, pts); ++cellid)
+    for(vtkIdType cellid = 0; input->GetLines()->GetNextCell(n, cellPts); ++cellid)
```

Comment:
I was completely perplexed by the old code's behavior w.r.t. to `pts` and by association the new code w.r.t. `cellPts`. In the old code, I kept looking for some place `pts` is either allocated or freed (or was thinking we might have a leak). But, it looks like what `GetNextCell` does is just update a pointer to some internal object data and it should not ever be modified/freed. Thats probably expedient from a performance perspective but threw me for a loop in reading this block of code.

Reply:
In this case I changed 'pts' to 'cellPts' because 'pts' is used again later in a different context.

Looking again at the documentation, this probably should have been changed to use vtkCellArrayIterator for VTK9, a better way of traversing the vtkCellArray (which is what input->GetLines() returns).  VTK 9 documentation states the method isn't thread safe.  I hadn't noticed that before.

Putting this PR in draft so I can reconsider the changes.

Reply:
Can't figure out how to put it in draft mode :(

Reply:
![Screen Shot 2022-08-12 at 2 05 51 PM](https://user-images.githubusercontent.com/5720676/184444945-d8984f51-b545-460c-a6b2-b7ccdc7602d7.png)